undertow/src/main/java/org/wildfly/extension/undertow/UndertowTransformers.java

```diff
@@ -118,13 +120,22 @@ private static void registerTransformersWildFly18(ResourceTransformationDescript
     }
 
     private static void registerTransformersWildFly16(ResourceTransformationDescriptionBuilder subsystemBuilder) {
+        subsystemBuilder.getAttributeBuilder()
+                .addRejectCheck(RejectAttributeChecker.DEFINED,OBFUSCATE_INSTANCE_ID)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, OBFUSCATE_INSTANCE_ID)
```

Comment:
Oops - my previous comment was incomplete. :(
We also want to discard if the attribute is defined, but set to the default value. i.e.
```java
.setDiscard(DiscardAttributeChecker.DEFAULT_VALUE, OBFUSCATE_INSTANCE_ID);
```
Same for other versions.

Reply:
@pferraro  Oh, I see. Please disregard my other comment about reverting the PR because of the test failure. I'll add the discard instead :-)